build/targets/standard-lifecycle.targets

```diff
@@ -1,17 +1,44 @@
 <Project>
-  <Target Name="Default" DependsOnTargets="Build" />
 
-  <Target Name="Build" DependsOnTargets="Clean;Initialize;Compile;Test;Verify;Publish" />
+  <PropertyGroup>
+    <Configuration>Debug</Configuration>
+  </PropertyGroup>
 
-  <Target Name="Clean" />
+  <ItemGroup>
+    <SolutionsToBuild Include="$(RepositoryRoot)\*.sln" />
```

Comment:
This won't work well at least for Mvc. Why not `$(RepositoryRoot)\$(RepoName).sln`?